packages/gatsby-source-wordpress/src/utils/gatsby-version.ts

```diff
@@ -1,5 +1,5 @@
-import semver from "semver"
-const gatsbyVersion = require(`gatsby/package.json`)?.version
+declare const _CFLAGS_: {
+  GATSBY_MAJOR: string
+}
```

Comment:
Is this still necessary with the tsconfig change?

Reply:
Yeah, it was throwing a ton of type errors when I extended the root's configs, and it wasn't pulling in the type correctly using `typeRoot`.

Reply:
👍